### PR TITLE
Lecture 02-b

### PR DESCRIPTION
**Pull request by viveky23**
Authors: tn1050@nyu.edu, vy404@nyu.edu, sk7685@nyu.edu.
Some implementation details of deep neural networks that enable us to get backpropagation algorithm work well. Designing and training a network using backprop requires making many implementation choices and we covered how to choose non-linearities, loss functions, minibatch sizes, shuffling strategies, input transformations, regularization and learning rate schedules. In addition.

We saw an illustration of the non-convex character of loss function of deep neural networks using a two layer neural network with hyperbolic tangent nonlinearity, which tries to approximate the identity function.

### CODE REVIEW

**Comment by Atcold** [comment]
Missing authors list in the PR description.
Please, check previously approved PR. Thanks.

**Comment by Atcold** [comment]
Why do they have same caption and label? This throws error. Did you compile the document?

```tex
\begin{figure}[ht]
\centering
\includegraphics[width=100mm]{lectures/02-b/Identity.PNG}
\caption{Gradient Descent vs. Stochastic Gradient Descent}
\label{fig:sgd}
\end{figure}
```
```tex
\begin{figure}[ht]
\centering
\includegraphics[width=120mm]{lectures/02-b/Stochastic.png}
\caption{Gradient Descent vs. Stochastic Gradient Descent}
\label{fig:sgd}
\end{figure}
```

Plz, fix. Thanks.

**Comment by viveky23** [comment]
Hey,
I have created a new pull request with the edited caption. I hope everything else is fine!

Thanks,
Vivek

**Comment by Atcold** [comment]
Labels are still wrong.
